### PR TITLE
Fix Image Adjustments Link & Link to Community Talk Video

### PR DESCRIPTION
**Pull request by SlRvb**
- Fixed the Image Adjustments Link going to a completely different snippet 🙈 
- Linked to the Community Talk iframe block instead of using another iframe to the same video

## Checklist

- [x] I have renamed all attached images with descriptive file names (or I did not include any images)
- [x] Before creating a new note, I searched the vault (or I only edited existing notes)
- [x] (Optional) In case I created a new note in the folder `04 - Guides, Workflows, & Courses`, I added a link to the new note(s) in one of the `for {group X}` overviews ([listed here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses)).

### CODE REVIEW

File: 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md [hunk 1]
```diff
@@ -26,8 +26,10 @@ Included in the [[ITS Theme]].
 ##### How to use the snippet:
 [Readme Guide](https://github.com/SlRvb/Obsidian--ITS-Theme#image-positions)
 
-**Video Explanation:** ["Slides" from the video](https://slrvb.github.io/Site/3_Nebula/ITS-Theme/ITST_Image-Positions/)
-<iframe width="100%" height="250px" src="https://www.youtube.com/embed/VRoBNWvw8sU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+**Video Explanation:**
```
Comment (chrisgrieser) [inline 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md:29]: but that's not a video though :D

File: 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md [hunk 2]
```diff
@@ -26,8 +26,10 @@ Included in the [[ITS Theme]].
...
+["Slides" from the video](https://slrvb.github.io/Site/3_Nebula/ITS-Theme/ITST_Image-Positions/)
```
Comment (chrisgrieser) [inline 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md:30]: about half of the embedded images on the GitHub Page seem to be dead links?
image

**Review by chrisgrieser — CHANGES_REQUESTED** [review]

File: 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md (hunk 1 above)
Comment (SlRvb) [reply 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md:29]: It is when you open the note 😉
Obsidian_cQIfQDysjW

File: 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md (hunk 2 above)
Comment (SlRvb) [reply 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md:30]: Oh right, I changed some of the titles and didn't update the site 🤦‍♀️. They should load up properly now.

File: 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md (hunk 1 above)
Comment (chrisgrieser) [reply 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md:29]: ah right, I always forgot about these embedded block references, my bad, sorry! 🙈

**Review by chrisgrieser — APPROVED** [review]

**Pull request merged**